scripts/dis_dump.py

```
@@ -0,0 +1,385 @@
+
+def _extract_instructions(code):
+    """Extract normalized instruction list from a code object.
+
+    - Filters out CACHE/PRECALL instructions
+    - Converts jump targets from byte offsets to instruction indices
+    - Resolves argument names via fallback when argrepr is missing
+    - Normalizes argument representations
+    """
+    try:
+        raw = list(dis.get_instructions(code))
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)